### PR TITLE
Add ScalaKata to Open Source Scala.js projects

### PR DESCRIPTION
**Pull request by MasseGuillaume**

### CODE REVIEW

File: community/index.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Use these Open Source projects to see how others have used Scala.js to build the
 - [Port of the Dart benchmark harness](https://github.com/sjrd/scalajs-benchmarks)
 - [2048 Game in Scala](https://github.com/fijolekProjects/scalajs-react-2048), 2048 game clone made using scalajs-react in just 350 lines of Scala and 250 lines of CSS.
 - [swimstats.org](http://www.swimstats.org/) ([source](https://github.com/andywag/swim_web)): Display/plot amateur swim results and times, by Andy Wagner
+- [ScalaKata](http://scalakata.com) ([source](https://github.com/MasseGuillaume/ScalaKata2):  Scala Interactive Playground
```
Comment (sjrd) [inline community/index.md]: There are two spaces instead of one after the `:`

**Comment by sjrd** [comment]
Otherwise LGTM.
(please *amend* your commit and force-push; do not push a second commit)

**Comment by MasseGuillaume** [comment]
@sjrd Done

**Comment by sjrd** [comment]
LGTM

**Pull request merged**